src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -97,12 +97,14 @@ export const PriceOptions: React.FC<PriceOptionsProps> = ({
   }
 
   const getPercentageOptions = () => {
-    return [0.2, 0.15, 0.1].map((pricePercentage, idx) => {
+    return [0, 0.1, 0.2].map((pricePercentage, idx) => {
       if (listPrice?.major) {
         return {
           key: `price-option-${idx}`,
           value: Math.round(listPrice.major * (1 - pricePercentage)),
-          description: `${pricePercentage * 100}% below the list price`,
+          description: !!pricePercentage
```

Comment:
non-blocking: I would not convert a number to boolean, I think it's clearer if compare `pricePercentage !== 0`

Reply:
Updated, thanks for the suggestion!